exercises/1-the-manual-menace/README.md

```diff
@@ -132,9 +111,10 @@ exit
 ```yaml
   hosts: "{{ target }}"
   vars:
-    ci_cd_namespace: donal-ci-cd
-    dev_namespace: donal-dev
-    test_namespace: donal-test
+    namespace_prefix: "Dónal"
```

Comment:
did you intentionally leave `Donal` here? :-)

Reply:
Not intentionally , seems to keep popping back up :) . Fixed